### PR TITLE
rm629 - added column error message info at ProcessesWithTasksPresenta…

### PR DESCRIPTION
**Pull request by ipyxhyp**
screenshot from 2018-03-21 06-29-19
[runawfe_development-629.pdf](https://github.com/processtech/runawfe-server/files/1884416/runawfe_development-629.pdf)

Column added to ProcessWithTasksPresentation form, updated resource struts properties.
Please review

### CODE REVIEW

**Comment by ipyxhyp** [comment]
At the moment - this new fix has not been tested on local environment.

**Comment by ipyxhyp** [comment]
This feature has not been tested in local environment running App yet.

File: wfe-core/src/main/java/ru/runa/wfe/execution/ProcessWithTasksClassPresentation.java [hunk 1]
```diff
@@ -138,6 +139,8 @@ private ProcessWithTasksClassPresentation() {
                         FieldFilterMode.DATABASE, "ru.runa.common.web.html.PropertyTDBuilder", new Object[] { new Permission(), "version" }),
                 new FieldDescriptor(PROCESS_EXECUTION_STATUS, String.class.getName(), new DefaultDBSource(Process.class, "executionStatus"), true,
                         FieldFilterMode.DATABASE, "ru.runa.wf.web.html.ProcessExecutionStatusTDBuilder", new Object[] {}),
+                new FieldDescriptor(PROCESS_EXECUTION_ERROR_MESSAGE, String.class.getName(), new DefaultDBSource(Process.class, "rootToken.errorMessage"), true,
```
Comment (dofs197) [inline wfe-core/src/main/java/ru/runa/wfe/execution/ProcessWithTasksClassPresentation.java]: Process relates many tokens. Root token error message is not sufficient.

**Comment by dofs197** [comment]
> Process relates many tokens. Root token error message is not sufficient.

**Pull request closed without merging**